blog/forms.py

```diff
@@ -0,0 +1,8 @@
+from django.forms import ModelForm
+
+from .models import Blog
+
+class BlogForm(ModelForm):
```

Comment:
nice, Django forms will save you code! :)

Reply:
Whoo hooo! :)